.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+
+# This runs every time a new commit is pushed to GitHub.
+on: [push]
```

Comment:
I recently learned this needs to be:
`on: [push, pull_request]`

In order for tests to run when folks create PRs in a fork!

Reply:
You can even use the following syntax:

```yml
push:
    branches: [main]
  pull_request:
    branches: [main]
```

and declare branches and patterns